make/ex_doc_wrapper.erl

```diff
@@ -0,0 +1,63 @@
+-module(ex_doc_wrapper).
+
+-export([main/1]).
+
+main(Args) ->
+    case os:find_executable("ex_doc") of
+        false ->  
+            case prompt() of
+                {ok, download} ->
+                    case download_otp() of
+                        error ->
+                            io:format("Unable to read ERL_TOP from environment?~n");
+                        ok ->
+                            main(Args)
+                    end;
+                {error, try_again} ->
+                    main(Args);
+                {error, do_not_download} ->
+                    io:format("Skipping download...~n")
```

Comment:
I probably need to error here otherwise `make` will continue running

Reply:
Confirmed and fixed in,

https://github.com/erlang/otp/pull/8490/commits/6a05b1fcf22ec2bf2a85e12d0ee7ab529bfdc1e5